python/diff-nautobot-understack/README.md

```diff
@@ -0,0 +1,34 @@
+# Nautobot vs OpenStack Comparison Tool
+
+This tool compares data between Nautobot and OpenStack.
+
+## Currently Supported Comparisons
+
+| OpenStack | Nautobot              |
+|-----------|-----------------------|
+| Project   | Tenant                |
+| Network   | UCVNI & Namespace     |
+
+
+---
+
+## Setup Instructions
+
+1. cd python/diff-nautobot-understack
+2. python3 -m venv .venv
+3. source .venv/bin/activate
+4. poetry lock
+5. poetry install
```

Comment:
```suggestion
2. poetry install
```

poetry will handle the creation of this virtual environment for you. It'll use `.venv` in the project if you configure it to do so locally on your machine with `poetry config virtualenvs.in-project true`.

Reply:
You should note that the user can create a shell with `poetry shell` and then when they exit it will clean up auth variables. Or they can run `source .venv/bin/activate` or `poetry run <commands-below>`